app/controllers/case_court_reports_controller.rb

```diff
@@ -59,12 +58,20 @@ def set_casa_case
     @casa_case = CasaCase.find_by(case_number: params[:id])
   end
 
+  def assigned_cases
+    @assigned_cases = if current_user.volunteer?
+                        CasaCase.actively_assigned_to(current_user)
+                      else
+                        current_user.casa_org.casa_cases.active
```

Comment:
I assumed here that we only wanted admin/supervisors to generate reports for active cases in their organization. Is that right?